features/steps/cli_steps.py

```diff
@@ -461,7 +461,7 @@ def check_one_node_run(context, number):
 
 @then('the logs should show that "{node}" was run')
 def check_correct_nodes_run(context, node):
-    expected_log_line = f"Running node: {node}"
+    expected_log_line = f"{node}"
```

Comment:
```suggestion
    expected_log_line = f"Running node: {node}"
```

And then `assert node in stdout` on L467. Let's carry on checking the whole `expected_log_line` in L471 because it still works.